### DISCUSSION TITLE
Is there a way to set the createdAt Date?

### DISCUSSION THREAD

**Discussion by stray-tori** [MarshalX/atproto#445] [Questions] [answered]
Before I went and made a suggestion, I first wanted to ask if I missed it.

**Question:** Does this SDK provide a way to set the creation date of a post?

**Context / Use Case** would be reposting some old content that I would want to set an older date for. The project i want to do this in is Python-based so I'd rather avoid swapping to the TS SDK if I can.

Thank you sm for anyone's time in advance!

**Comment by MarshalX** [comment, marked as answer]
copy this and put any `created_at` what you want 
https://github.com/MarshalX/atproto/blob/d030829488ab34ca4e601021e18ec2191f1f77ff/packages/atproto_client/client/client.py#L165-L173

**Reply by stray-tori** [reply]
OHH- that makes so much sense.
thank you so much! I just checked the documentation but didn't think to check the code of the functions A-
